front/app/components/Form/Components/Layouts/CLCategoryLayout.tsx

```diff
@@ -9,9 +9,13 @@ import { JsonFormsDispatch, withJsonFormsLayoutProps } from '@jsonforms/react';
 import { Box, fontSizes, media } from '@citizenlab/cl2-component-library';
 import { FormSection } from 'components/UI/FormComponents';
 import styled, { useTheme } from 'styled-components';
-import { FormElement } from 'components/IdeaForm';
 import QuillEditedContent from 'components/UI/QuillEditedContent';
 
+export const FormElement = styled.div`
+  width: 100%;
+  margin-bottom: 40px;
+`;
+
```

Comment:
Could we use a Box instead of a styled component here?

Reply:
Good idea!